### ISSUE TITLE
Replay, Repeat

### ISSUE THREAD

**Issue by SeasonalFerret** [Just-Some-Bots/MusicBot#1590]
Please tick all applicable boxes.

- [x] I am using Python 3.5 or higher (run `python --version` on the command line)
- [x] I have followed the [official guides](https://github.com/Just-Some-Bots/MusicBot/wiki) to install the bot for my system
- [x] I have updated my dependencies to the latest version using the appropriate `update` script

Which version are you using?

- [x] The latest `master` version (1.9.7)
- [ ] The latest `review` version

What type of issue are you creating?

- [ ] Bug
- [x] Feature request
- [ ] Question

----

### Feature request

Simple !replay command that adds the currently playing song to the bottom of the queue. 
Saves a little bit of time instead of re-!play'ing the same song over and over.

And a !repeat command that continuously plays the same song over and over until someone else adds something to the queue, then goes back to autoplaylist.txt. 
Saves a little bit of time instead of !replay'ing over and over.

**Comment by MattBSG** [comment]
Similarly, there is an issue for looping a song open as well. I'll mention is for convenience #128

**Comment by SeasonalFerret** [comment]
Noted, apologies.

**Comment by MattBSG** [comment]
I mean't as it is similar to your request. Looping a song plays it over and over again (like adding it back to the queue). What you are asking is slightly different however

**Comment by SeasonalFerret** [comment]
Simply referencing this 'issue' in the previous was good enough, but if you feel that this is different enough to keep as a separate issue then feel free.

**Comment by AutumnClove** [comment]
Repeat #1677

**Comment by AutumnClove** [comment]
Closing because this has been implemented and is waiting on a merge.

**Comment by ishowta** [comment]
It didn't merge.
